apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -1121,6 +1121,8 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
     }
 
     public applyAdvanceSearch(request: InvoiceFilterClassForInvoicePreview) {
+        request.from = this.invoiceSearchRequest.from,
+        request.to = this.invoiceSearchRequest.to
         this.showAdvanceSearchIcon = true;
         if (!request.invoiceDate && !request.dueDate || this.selectedVoucher === VoucherTypeEnum.purchase) {
```

Comment:
Can we remove this condition as the `request.from` and `request.to` is already assigned on line number 1128. **BUT PLEASE TEST IT PROPERLY ON LOCAL** before making changes to PR as it should not break any other flow. Do test it for all the voucher types